### PR TITLE
Format all Python code with `black`

### PR DESCRIPTION
**Pull request by mntns**
This formats all the Python code using [black](https://github.com/psf/black), making it more consistent, while being PEP 8 compliant. 

I think for the future it could be nice to explicitly use and mention the `black` code style in a contribution guide and check for that in CI runs, related to #106.

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -6,6 +6,6 @@
```
Comment (dedekind) [inline __main__.py:6]: Just a note: I believe we need to switch for 80 chars per line to 100.

And we use this header now:
`# vim: ts=4 sw=4 et ai si
`
Instead, we should use this:

`# vim: ts=4 sw=4 tw=100 et ai si
`
Would you care enough to do this too? Thanks in advance!

File: bmaptools/BmapCreate.py [hunk 2]
```diff
@@ -155,8 +155,9 @@ def __init__(self, image, bmap, chksum_type="sha256"):
         try:
             self._cs_len = len(hashlib.new(self._cs_type).hexdigest())
         except ValueError as err:
-            raise Error("cannot initialize hash function \"%s\": %s" %
-                        (self._cs_type, err))
+            raise Error(
+                'cannot initialize hash function "%s": %s' % (self._cs_type, err)
+            )
```
Comment (dedekind) [inline bmaptools/BmapCreate.py:160]: This sort of change is controversial, but ... I do not like it personally, but I also think current code needs lots of clean-ups. So anyway, overall it is better to follow "black" suggestions. Thanks!

**Review by dedekind — APPROVED** [review]

**Comment by dedekind** [comment]
Thanks!

File: __main__.py (hunk 1 above)
Comment (mntns) [reply __main__.py:6]: Following the [black code style](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length), I set the `textwidth` to `88`.

File: bmaptools/BmapCreate.py (hunk 2 above)
Comment (mntns) [reply bmaptools/BmapCreate.py:160]: I agree in not liking some of these formatting changes, but that it makes sense for now, for gaining consistency and adhering to a certain code style for further clean-ups.

**Pull request merged**